services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.tsx

```diff
@@ -43,27 +47,50 @@ export const ReleaseCard: React.FC<ReleaseCardProps> = (props) => {
     }, []);
 
     return (
-        <div className={classNames('mdc-card release-card', className)} onClick={clickHandler}>
-            <div className="release-card__header">
-                <div className="release__title mdc-typography--headline6">{sourceMessage}</div>
-                {!!sourceCommitId && <Button className="release__hash" label={sourceCommitId} />}
-            </div>
-            <div className="mdc-card__primary-action release-card__description" ref={control} tabIndex={0}>
-                <div className="mdc-card__ripple" />
-                <div className="release__details">
-                    {!!createdAt && (
-                        <div className="release__metadata mdc-typography--subtitle2">
-                            <div>{'Created at: ' + createdAt.toLocaleDateString()}</div>
-                            <div>{'Time ' + createdAt.toLocaleTimeString()}</div>
-                        </div>
-                    )}
-                    <div className="release__version mdc-typography--body2">{'Version: ' + version}</div>
-                    <div className="release__author mdc-typography--body1">{'Author: ' + sourceAuthor}</div>
-                </div>
+        <Tooltip
+            id={app + version}
+            content={
+                <>
+                    <h2 className="mdc-tooltip__title release__details">
+                        {!!sourceMessage && <b>{sourceMessage}</b>}
+                        {!!sourceCommitId && <Button className="release__hash" label={sourceCommitId} />}
+                        {!!sourceAuthor && <div>{'| ' + sourceAuthor + ' |'}</div>}
+                        {!!createdAt && (
+                            <div className="release__metadata mdc-typography--subtitle2">
+                                <div>
+                                    {`${createdAt.getDay()}-${createdAt.getMonth()}-${createdAt.getFullYear()}` +
+                                        ' @ ' +
+                                        `${createdAt.getHours()}:${createdAt.getMinutes()}` +
+                                        ' | '}
+                                    <i>
+                                        {daysToString(((Date.now().valueOf() - createdAt.valueOf()) / MsPerDay) >> 0)}
```

Comment:
Nit: Similar here:
can we put that into a variable or function?

Reply:
my goal here is to give it a name, then it's easier to understand.
I actually don't understand what `/ MsPerDay >> 0` does.
Can we please avoid the shift operator?

Reply:
Also done! 
I forgot about the floor function and couldn't think of it :grimacing: